CONTRIBUTING.md

```diff
@@ -1,16 +1,16 @@
 # Contributor's Guide
 
 'HOW TO CONTRIBUTE TO OPEN SOURCE' accepts PR's (pull requests) from **newbies**
-only, this is to help **newbies** get familiar with the contribution processes.
+only. This is to help **newbies** get familiar with the contribution processes.
 
-Issues can be submitted by anyone - seasoned developers or newbies.
+Issues can be submitted by anyone - either seasoned developers or newbies.
 
 **Contents**
 
 - [Getting Started](#getting-started)
 - [Submitting a Pull Request](#submitting-a-pull-request)
 - [Adding to the Main README](#adding-to-the-main-readme)
-- [Adding to Non-English README](#adding-to-non-english-readme)
+- [Adding to Non-English README](#adding-to-non-English-readme)
```

Comment:
This isn't necessary, as this is simply a link.

```suggestion
- [Adding to Non-English README](#adding-to-non-english-readme)
```

Reply:
Done.